.gitlab-ci.yml

```diff
@@ -83,6 +136,7 @@ nightly-test-notification:
   needs:                           ["nightly-test"]
   when:                            always
   script:
+    - TEST_JOB_STATUS=$(<nightly_test_result)
```

Comment:
Added here, because threw away by mistake in https://github.com/paritytech/staking-miner-v2/pull/145